rednotebook/external/text2rednotebook.py

```diff
@@ -0,0 +1,145 @@
+"""
```

Comment:
Can you move the script to a new "scripts/" folder at the root of the repo, please?

Reply:
Done